src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -511,6 +531,21 @@ public YesNo getRegulation35Selection() {
         }
     }
 
+    @JsonIgnore
+    public List<String> getSchedule7Selections() {
+        if (sscsUcCaseData != null) {
+            if ("Yes".equalsIgnoreCase(sscsUcCaseData.getUcWriteFinalDecisionSchedule7ActivitiesApply())) {
+                return sscsUcCaseData.getUcWriteFinalDecisionSchedule7ActivitiesQuestion();
+            } else if ("No".equalsIgnoreCase(sscsUcCaseData.getUcWriteFinalDecisionSchedule7ActivitiesApply())) {
+                return new ArrayList<>();
+            } else {
+                return null;
+            }
+        } else {
+            return null;
+        }
```

Comment:
These last two else statements could be removed by having a return null at the bottom of the method. There could be less indentation if the getter method for sscsUcCaseData is used to remove the null check

Reply:
Yes, I've changed it - much cleaner